docs/docs/schema-customization.md

```diff
@@ -319,6 +319,10 @@ Gatsby's automatic type inference has one trick up its sleeve: for every field
 that ends in `___NODE` it will interpret the field value as an `id` and create a
 foreign-key relation.
 
+> Note: Before the introduction of the Schema Customization APIs in Gatsby v2.2,
```

Comment:
I'm unsure what the takeaway is here - as a reader should this inform me that using mappings or `___NODE` are equivalent, and that in both cases I can now use schema customisation instead?

Reply:
yes -- previously a plugin author would use the `___NODE` naming convention to create links between nodes when *creating the nodes* (i.e. used by source/transformer plugins), and a user would use the `mappings` config key. both users and plugin authors can now use the `@link` extension.

Reply:
That's great - I added one more line at the end that matches your comment here